Source/App/EbAppMain.c

```diff
@@ -75,6 +75,95 @@ void AssignAppThreadGroup(uint8_t targetSocket) {
 #endif
 }
 
+void* EbAppCreateThread(
+    void *threadFunction(void*),
+    void *threadContext)
+{
+    void* threadHandle = NULL;
+
+#ifdef _WIN32
+    threadHandle = (void*)CreateThread(
+        NULL,                           // default security attributes
+        0,                              // default stack size
+        (LPTHREAD_START_ROUTINE)threadFunction, // function to be tied to the new thread
+        threadContext,                  // context to be tied to the new thread
+        0,                              // thread active when created
+        NULL);                          // new thread ID
+#else
+    pthread_attr_t attr;
+    struct sched_param param = {
+        .sched_priority = 99
```

Comment:
Some thoughts about the 99 priority
1) Make this value a #define rather than a hardcoded value
2) http://man7.org/linux/man-pages/man7/sched.7.html shows the range as -19 to 20 and mentions that values outside that are clamped.  Perhaps change to 20?

Reply:
```
       For each of the above policies, param->sched_priority specifies a
       scheduling priority for the thread.  This is a number in the range
       returned by calling sched_get_priority_min(2) and
       sched_get_priority_max(2) with the specified policy.  On Linux, these
       system calls return, respectively, 1 and 99.
```
Simply copied from svt library.